src/lib/utils/formatter/percentage.ts

```diff
@@ -19,3 +21,6 @@ export const formatPercentValue = (value: Big): string => {
   }
   return `${d6}%`;
 };
+
+export const formatRatioToPercent = (ratio: Ratio<BigSource>): string =>
```

Comment:
Simply name it `formatRatio` and include a comment explaning that this function is used to format Ratio to display string (as percentage)